### PR TITLE
Update top-level `CHANGELOG.md` with cardano-node relevant changes

### PR DESCRIPTION
**Pull request by lehins**
# Description

Ledger team still needs to provide high level overview of the changes that go into each major cardano-node release. Might as well record them in the cardano-ledger repository as well.

# Checklist

- [ ] Commit sequence broadly makes sense and commits have useful messages
- [ ] New tests are added if needed and existing tests are updated
- [ ] When applicable, versions are updated in `.cabal` and `CHANGELOG.md` files according to the
      [versioning process](https://github.com/input-output-hk/cardano-ledger/blob/master/RELEASING.md#versioning-process).
- [ ] The version bounds in `.cabal` files for all affected packages are updated. **If you change the bounds in a cabal file, that package itself must have a version increase.** (See [RELEASING.md](https://github.com/input-output-hk/cardano-ledger/blob/master/RELEASING.md#versioning-process))
- [ ] All visible changes are prepended to the latest section of a `CHANGELOG.md` for the affected packages. **New section is never added with the code changes.** (See [RELEASING.md](https://github.com/input-output-hk/cardano-ledger/blob/master/RELEASING.md#changelogmd))
- [x] Code is formatted with [`fourmolu`](https://github.com/fourmolu/fourmolu) (use `scripts/fourmolize.sh`)
- [x] Cabal files are formatted (use `scripts/cabal-format.sh`)
- [x] [`hie.yaml`](https://github.com/input-output-hk/cardano-ledger/blob/master/hie.yaml) has been updated (use `scripts/gen-hie.sh`)
- [x] Self-reviewed the diff

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,52 @@
...
 
-**THIS CHANGELOG HAS BEEN DEPRECATED**
+This changelog contains notable changes that make into each [`cardano-node` release](https://github.com/input-output-hk/cardano-node/releases),
+starting with version `8.0`. We also provide per-Haskell-package changelogs. Please see
+the `CHANGELOG.md` for each individual package for any changes relevant for developers.
 
-We have adopted per-Haskell-package changelogs. Please see the `CHANGELOG.md` for each
-individual package for any notable changes.
+If you are looking for the Ledger Releasing and Versioning Process then you can find it in
+[REALEASING.md](https://github.com/input-output-hk/cardano-ledger/blob/master/RELEASING.md#changelogmd).
```
Comment (teodanciu) [inline CHANGELOG.md]: small typo  "REALEASING" -> "RELEASING" 
(though  "releasing" is a pretty cool idea hehe)

Comment (lehins) [reply CHANGELOG.md]: Thank you. Fixed

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,11 +1,52 @@
 # Changelog
 
-**THIS CHANGELOG HAS BEEN DEPRECATED**
+This changelog contains notable changes that make into each [`cardano-node` release](https://github.com/input-output-hk/cardano-node/releases),
```
Comment (teodanciu) [inline CHANGELOG.md]: This is perfectly  understandable, but maybe "changes that made it into"  or "that make it into" would sound a little more neat?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,11 +1,52 @@
...
-individual package for any notable changes.
+If you are looking for the Ledger Releasing and Versioning Process then you can find it in
+[RELEASING.md](https://github.com/input-output-hk/cardano-ledger/blob/master/RELEASING.md#changelogmd).
 
-See the [REALEASING.md](https://github.com/input-output-hk/cardano-ledger/blob/master/RELEASING.md#changelogmd) for an up-to-date information about Releasing and Versioning Process.
+## 8.2.0
+
+- Restructure certificate interface
+- Conway related changes:
+  - Introduction of a `DRep`, with the default ones that vote No and Abstain.
```
Comment (teodanciu) [inline CHANGELOG.md]: Another nitpick:  "Introduce `DRep` with the defaults"  would be more consistent with the list below, but it's really just aesthetics.

**Review by teodanciu — APPROVED** [review]
Looks good to me, nice to have this!

File: CHANGELOG.md (hunk 2 above)
Comment (lehins) [reply CHANGELOG.md]: Funny that you correct me there, cause it was `made`, but I changed it to `make` :smile: 

Thank you for correcting me back! Fixed.

File: CHANGELOG.md (hunk 3 above)
Comment (lehins) [reply CHANGELOG.md]: Perfect. Done

**Pull request merged**